app/page.tsx

```diff
@@ -1169,7 +1232,15 @@ export default function Home() {
                   </a>
                 </div>
               </div>
-            </div>
+            </div>{/* 右下にAIチャットボットのアイコンを設置 */}
+            <button
+              className="fixed bottom-8 right-8 z-40 bg-gradient-to-r from-blue-500 to-purple-500 text-white rounded-full shadow-lg w-16 h-16 flex items-center justify-center text-3xl hover:scale-105 transition"
+              onClick={openAiChat}
+              aria-label="AIチャットを開く"
+              style={{ boxShadow: "0 4px 24px rgba(80, 80, 200, 0.18)" }}
+            >
+              🤖
+            </button>
```

Comment:
コピペで２回登場してるかな？
そういうときはcomponentsに分けてimportしましょう。
例えばaiChatButtonとかで切り出せば
```suggestion
<aiChatButton>
```
きれいになります。1169-1176行目もこれでOK。

Reply:
9295a1992302fcfc62097021e2405738dfd371e2
cb622db2a73f50dbfb92f73639acde4724d3d584